src/cim/components/ClusterDeployment/ClusterDeploymentNetworkingStep.tsx

```diff
@@ -101,7 +101,7 @@ const ClusterDeploymentNetworkingStep: React.FC<ClusterDeploymentDetailsNetworki
     } catch (error) {
       addAlert({
         title: 'Failed to save ClusterDeployment',
-        message: error,
+        message: error.message as string,
```

Comment:
`onSaveNetworking` throws `Error` object so it is correct in this case. However, since the eslint rule forces the error type in 'catch' clause to be 'unknown' type, we need to check the error before setting the message. An utility such as following would be nice.
```
function getErrorMessage(error, fallbackMessage = '') {
  if (error instanceof Error) {
    return error.message;
  }
  return fallbackMessage;
}